wrappers/vcx-napi-rs/Cargo.toml

```diff
@@ -1,9 +1,11 @@
 [package]
 name = "vcx-napi-rs"
-version.workspace = true
-authors.workspace = true
-license.workspace = true
-edition.workspace = true
+version = "1.0.0"
+edition = "2021"
+# version.workspace = true
```

Comment:
not sure about the commented config here

Reply:
This should not have been committed of course, made some changes to appease LSP